climpred/metrics.py

```diff
@@ -39,26 +39,100 @@ def _get_norm_factor(comparison):
 
 # wrap xskillscore metrics to work with comparison argument
 def _pearson_r(forecast, reference, dim='svd', comparison=None):
+    """
+    Calculate the Anomaly Correlation Coefficient (ACC).
+
+    .. math::
+        ACC = \\frac{cov}{std std}
+
+    Range:
+        * perfect: 1
+        * min: -1
+
+    See also:
+        * xskillscore.pearson_r
+        * xskillscore.pearson_r_p_value
+    """
     return pearson_r(forecast, reference, dim=dim)
 
 
 def _pearson_r_p_value(forecast, reference, dim='svd', comparison=None):
+    """
+    Calculate the probability associated with the ACC being not random?"""
     return pearson_r_p_value(forecast, reference, dim=dim)
 
 
 def _mse(forecast, reference, dim='svd', comparison=None):
+    """
+    Calculate the Mean Sqaure Error (MSE).
+
+    .. math::
+        MSE = \overline{(f - o)^{2}}
+
+    Range:
+        * perfect: 0
+        * min: 0
+        * max: \\inf
+
+    See also:
+        * xskillscore.mse
+    """
     return mse(forecast, reference, dim=dim)
 
 
 def _rmse(forecast, reference, dim='svd', comparison=None):
+    """
+    Calculate the Root Mean Sqaure Error (RMSE).
+
+    .. math::
+        RMSE = \sqrt{\overline{(f - o)^{2}}}
+
+    Range:
+        * perfect: 0
+        * min: 0
+        * max: \\inf
+
+    See also:
+        * xskillscore.rmse
+    """
     return rmse(forecast, reference, dim=dim)
 
 
 def _mae(forecast, reference, dim='svd', comparison=None):
+    """
+    Calculate the Mean Absolute Error (MAE).
+
+    .. math::
+        MSE = \overline{(f - o)^{2}}
+
+    Range:
+        * perfect: 0
+        * min: 0
+        * max: \\inf
+
+    See also:
+        * xskillscore.mae
+    """
     return mae(forecast, reference, dim=dim)
 
 
 def _crps(forecast, reference, dim='svd', comparison=None):
+    """
+    Continuous Ranked Probability Score.
+
+    References:
```

Comment:
Would be good to have Range above References here and in other places for consistency

Reply:
I dont really understand CRPS yet. Thats why information is sparse there.